toolkit/tools/imagegen/diskutils/diskutils.go

```diff
@@ -401,7 +401,7 @@ func FormatSinglePartition(partDevPath string, partition configuration.Partition
 func SystemBlockDevices() (systemDevices []SystemBlockDevice, err error) {
 	var blockDevices blockDevicesOutput
 
-	rawDiskOutput, stderr, err := shell.Execute("lsblk", "-d", "--bytes", "-I", "8,179,259", "-n", "--json", "--output", "NAME,SIZE,MODEL")
+	rawDiskOutput, stderr, err := shell.Execute("lsblk", "-d", "--bytes", "-I", "8,179,252,253,254,259", "-n", "--json", "--output", "NAME,SIZE,MODEL")
```

Comment:
Change looks fine. I was wondering if we could also add a comment explaining what each number (or number range) is needed for.

Reply:
I think that since this is a call out to another program, one can find determine the options via the `lsblk` man page.

Reply:
I was more focused on the "why" part when I mentioned the comment. My main concern is that it's not obvious why these and only these numbers are used.

Reply:
Added consts to clarify the numbers.